aptos-move/framework/aptos-stdlib/sources/cryptography/groups.move

```diff
@@ -0,0 +1,744 @@
+module aptos_std::groups {
```

Comment:
We should see if our Move docs support LaTeX. 

If so, we should use it.

Reply:
Yes it does!